drivers/bus/acpi/main.c

```diff
@@ -359,6 +383,7 @@ AcpiRegOpenKey(IN HANDLE ParentKeyHandle,
 }
 
 static
+CODE_SEG("INIT")
```

Comment:
These registry helpers are only used during INIT? hm weird...

Reply:
These registry helpers are not used anywhere else besides this file.